CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 New functionality that are backwards-compatible:
 * [OLMIS-7373](https://openlmis.atlassian.net/browse/OLMIS-7373): Add filtering by product code, name and lot code on Stock on Hand page
+* [OLMIS-7594](https://openlmis.atlassian.net/browse/OLMIS-7594): Add possibility to use the same Lot Code in two or more different products
```

Comment:
I would move it to bugs and rename to "Fix issue with adding same lot codes for different products in one physical inventory draft". Users were able to use same lot codes to different products before so it doesn't describe the main cause of the problem which was to add these in one draft.

Reply:
fixed as you suggested